### PR TITLE
Replace built-in user model with get_user_model

### PR DESCRIPTION
**Pull request by cody-lofty**
Fixes issue described in #173

### CODE REVIEW

File: testproject/requirements.txt [hunk 1]
```diff
@@ -19,3 +19,4 @@ coverage==5.5
 
 isort
 flake8
+flaky
```
Comment (jhurej) [inline testproject/requirements.txt]: what is that for? I mean, I know what's that library but you don't use it anywhere in PR

Comment (jhurej) [reply testproject/requirements.txt]: nvm, I see it's used in tests but not added on the develop branch you were aligned to

**Comment by jhurej** [comment]
@cody-lofty looks good, I tested it with a custom User model without a username field and it's working fine. Please rebase to development branch and then I'll merge it

File: testproject/requirements.txt (hunk 1 above)
Comment (cody-lofty) [reply testproject/requirements.txt]: apologies, side effect of setting up the project 👍

**Comment by cody-lofty** [comment]
Rebased to develop!

**Review by jhurej — APPROVED** [review]
lgtm

**Pull request merged**